ocfweb/api/session_tracking.py

```diff
@@ -0,0 +1,108 @@
+import json
+from functools import partial
+from ipaddress import ip_address
+
+from django.conf import settings
+from django.http import HttpResponse
+from django.http import HttpResponseBadRequest
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_POST
+from ocflib.infra.hosts import hosts_by_filter
+from ocflib.infra.net import is_ocf_ip
+from ocflib.lab.stats import get_connection
+
+from ocfweb.caching import cache
+
+get_connection = partial(
+    get_connection,
+    user=settings.OCFSTATS_USER,
+    password=settings.OCFSTATS_PASSWORD,
+    db=settings.OCFSTATS_DB,
+)
+
+
+@require_POST
+@csrf_exempt
+def log_session(request):
+    """Primary API endpoint for session tracking.
+
+    Desktops have a cronjob that calls this endpoint, replacing
+    the functionality that used to be in ocf/labstats.
+    """
+
+    remote_ip = request.META['REMOTE_ADDR']
```

Comment:
I think you'll want to use `django-ipware` like we do in the [vhost request form](https://github.com/ocf/ocfweb/blob/6aa0e1ce31497358f153ee29527a603b706a40de/ocfweb/account/vhost.py#L74) to get the user's real IP instead of getting the IP of the proxy host. If that doesn't work, you'd probably want to use [Yelp/wsgi-mod-rpaf](https://github.com/yelp/wsgi-mod-rpaf) or something with some more work to get all the proxy layers between to pass the correct headers. We'd probably want to do this anyway some day so that we can get a good idea of which IPs are actually hitting ocfweb from inside the containers themselves, and having this as a middleware removes the duplication that having to retrieve the correct IP everywhere individually would include.